paasta_tools/kubernetes_tools.py

```diff
@@ -1139,6 +1150,26 @@ def __init__(self) -> None:
         self.autoscaling = kube_client.AutoscalingV2beta1Api()
 
 
+def force_delete_pods(
+    service: str,
+    paasta_service: str,
+    instance: str,
+    namespace: str,
+    kube_client: KubeClient,
+):
+    all_pods = pods_for_service_instance(paasta_service, instance, kube_client)
+    pods_to_delete = filter_pods_by_service_instance(all_pods, paasta_service, instance)
+    delete_options = V1DeleteOptions()
+    kube_client.deployments.delete_namespaced_deployment(
+        service, namespace, body=delete_options
+    )
+    delete_options.grace_period_seconds = 0
```

Comment:
I don't think you need to set this.

Reply:
This is None otherwise, so I wanted to make sure it it set to 0.